docs/global.html

```
@@ -0,0 +1,2446 @@
+
+
+
+
+
+        <h4 class="name" id="p2pAddNodes"><span class="type-signature"></span>p2pAddNodes<span class="type-signature"> :Array.&lt;string></span></h4>
+
+
+
+
+        <div class="description">
+          After RPC is ready, call `addnode <host:port> add` for each entry (outbound P2P only).
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </host:port> ], start tag match failed [ <host:port> ] on line 530.

(tag-pair)